index.html

```diff
@@ -42,6 +42,7 @@
         <li><a href="#stingray">Cell Site Simulators</a></li>
         <li><a href="#shotspotter">Shotspotter</a></li>
         <li><a href="#socmint">Social Media</a></li>
+		<li><a href="#cvrm">Predictive Policing</a></li>
```

Comment:
I think this indent shouldn't be here?

Reply:
That's odd. I don't see an indent on my local version.